WebApp/Web/shared.py

```diff
@@ -358,25 +358,29 @@ def doner_tissue_to_js(data):
     unit_type = None
     for item in data:
 
-        if not ("donor" in item["object"]["value"].lower() or "tissuesample" in item["object"]["value"].lower()):
-            key = split_and_extract_last(item["property"]["value"])
-            value = item["object"]["value"]
-
-            if key == "Label":
-                key = "Local Name"
-            elif key == "Was Derived From":
-                key = "Source"
-                value = extract_uri_data(item["object"]["value"])
-            elif key == "Species":
-                key = "Taxon Number"
-            elif key == "Age  At  Death  Unit":
-                unit_type = value
-            elif key == "Age  At  Death  Value":
-                value = f"{int(float(value))} {unit_type}"
-            elif key == "Biological  Sex":
-                value = _sex_int_to_word(value)
-
-            js_data.append({key: value})
+        try:
+            if not ("donor" in item["object"]["value"].lower() or "tissuesample" in item["object"]["value"].lower()):
+                key = split_and_extract_last(item["property"]["value"])
+                value = item["object"]["value"]
+
+                if key == "Label":
+                    key = "Local Name"
+                elif key == "Was Derived From":
+                    key = "Source"
+                    value = extract_uri_data(item["object"]["value"])
+                elif key == "Species":
+                    key = "Taxon Number"
+                elif key == "Age  At  Death  Unit":
+                    unit_type = value
+                elif key == "Age  At  Death  Value":
+                    value = f"{int(float(value))} {unit_type}"
+                elif key == "Biological  Sex":
+                    value = _sex_int_to_word(value)
```

Comment:
@aaronkanzer please don't review `WebApp/Web/shared.py`. Didn't realize it got pushed as this is obsolete. I had just kept it there so that I could see the old version. The `WebApp` has been changed to use `NextJS` for UI. Sorry, for this push, should have checked.